wallet/wallet.c

```diff
@@ -1620,33 +1608,28 @@ void wallet_channel_save(struct wallet *w, struct channel *chan)
 	db_bind_amount_sat(stmt, 12, &chan->funding);
 	db_bind_amount_sat(stmt, 13, &chan->our_funds);
 	db_bind_int(stmt, 14, chan->remote_funding_locked);
-	db_bind_int(stmt, 15, chan->remote_tx_sigs);
-	db_bind_amount_msat(stmt, 16, &chan->push);
-	db_bind_amount_msat(stmt, 17, &chan->our_msat);
-
-	db_bind_talarr(stmt, 18, chan->shutdown_scriptpubkey[REMOTE]);
-	db_bind_u64(stmt, 19, chan->final_key_idx);
-	db_bind_u64(stmt, 20, chan->our_config.id);
-	db_bind_psbt(stmt, 21, chan->last_tx->psbt);
-	db_bind_signature(stmt, 22, &chan->last_sig.s);
-	db_bind_int(stmt, 23, chan->last_was_revoke);
-	db_bind_int(stmt, 24, chan->min_possible_feerate);
-	db_bind_int(stmt, 25, chan->max_possible_feerate);
-	db_bind_amount_msat(stmt, 26, &chan->msat_to_us_min);
-	db_bind_amount_msat(stmt, 27, &chan->msat_to_us_max);
-	db_bind_int(stmt, 28, chan->feerate_base);
-	db_bind_int(stmt, 29, chan->feerate_ppm);
-	db_bind_talarr(stmt, 30, chan->remote_upfront_shutdown_script);
-	db_bind_int(stmt, 31, chan->option_static_remotekey);
-	db_bind_int(stmt, 32, chan->option_anchor_outputs);
-	db_bind_talarr(stmt, 33, chan->shutdown_scriptpubkey[LOCAL]);
-	if (chan->psbt)
-		db_bind_psbt(stmt, 34, chan->psbt);
-	else
-		db_bind_null(stmt, 34);
-	db_bind_int(stmt, 35, chan->closer);
-	db_bind_int(stmt, 36, chan->state_change_cause);
-	db_bind_u64(stmt, 37, chan->dbid);
+	db_bind_amount_msat(stmt, 15, &chan->push);
+	db_bind_amount_msat(stmt, 16, &chan->our_msat);
+
+	db_bind_talarr(stmt, 17, chan->shutdown_scriptpubkey[REMOTE]);
+	db_bind_u64(stmt, 18, chan->final_key_idx);
+	db_bind_u64(stmt, 19, chan->our_config.id);
+	db_bind_psbt(stmt, 20, chan->last_tx->psbt);
+	db_bind_signature(stmt, 21, &chan->last_sig.s);
+	db_bind_int(stmt, 22, chan->last_was_revoke);
+	db_bind_int(stmt, 23, chan->min_possible_feerate);
+	db_bind_int(stmt, 24, chan->max_possible_feerate);
+	db_bind_amount_msat(stmt, 25, &chan->msat_to_us_min);
+	db_bind_amount_msat(stmt, 26, &chan->msat_to_us_max);
+	db_bind_int(stmt, 27, chan->feerate_base);
+	db_bind_int(stmt, 28, chan->feerate_ppm);
+	db_bind_talarr(stmt, 29, chan->remote_upfront_shutdown_script);
+	db_bind_int(stmt, 30, chan->option_static_remotekey);
+	db_bind_int(stmt, 31, chan->option_anchor_outputs);
+	db_bind_talarr(stmt, 32, chan->shutdown_scriptpubkey[LOCAL]);
+	db_bind_int(stmt, 33, chan->closer);
+	db_bind_int(stmt, 34, chan->state_change_cause);
+	db_bind_u64(stmt, 35, chan->dbid);
 	db_exec_prepared_v2(take(stmt));
```

Comment:
Should this commit remove those fields from the db migrations as well?

Reply:
removing fields from the db (in sqlite) requires 
- copy existing table to temp table
- deleting existing table
- re-creating table without fields
- copying data back from temp table
- deleting temp table

Doable, but probably best to execute in a separate PR with extensive tests.